build

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+mvn clean install -DunofficialSuffix=
```

Comment:
The "official" prefix should be used by the release build task only.
It may make sense to forward parameters given to this `build` script to the `mvn` command like this:
`mvn clean install "$@"`